burstphoto/App.swift

```diff
@@ -21,6 +21,30 @@ class AppDelegate: NSObject, NSApplicationDelegate {
         // ensure that tabs cannot be created
         // let _ = NSApplication.shared.windows.map { $0.tabbingMode = .disallowed }
         NSWindow.allowsAutomaticWindowTabbing = false
+        
+        // create output directory
+        if !FileManager.default.fileExists(atPath: NSHomeDirectory() + "/Pictures/Burst Photo/") {
+            do {
+                try FileManager.default.createDirectory(atPath: NSHomeDirectory() + "/Pictures/Burst Photo/", withIntermediateDirectories: true, attributes: nil)
+            } catch {}
+        }
+        
+        do {
+            // If it exists, delete a previously leftover temporary directory
+            var isDirectory : ObjCBool = true
+            if FileManager.default.fileExists(atPath: NSHomeDirectory() + "/Pictures/Burst Photo/.dngs/", isDirectory: &isDirectory) {
+                try FileManager.default.removeItem(atPath: NSHomeDirectory() + "/Pictures/Burst Photo/.dngs/")
+            }
+            try FileManager.default.createDirectory(atPath: NSHomeDirectory() + "/Pictures/Burst Photo/.dngs/", withIntermediateDirectories: true)
+        } catch {}
```

Comment:
This ideally uses AppStorage in AppSettings so that these folders can be user specified (or at least not hard coded in here), but I couldn't figure how to get access to that from inside one of the functions that get called on startup.